cloudflare/resource_cloudflare_byo_ip_prefix_test.go

```diff
@@ -0,0 +1,46 @@
+package cloudflare
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccCloudflareBYOIPPrefix(t *testing.T) {
+	t.Parallel()
+	prefixID := os.Getenv("CLOUDFLARE_BYO_IP_PREFIX_ID")
+
+	rnd := generateRandomResourceName()
+	name := fmt.Sprintf("cloudflare_byo_ip_prefix.%s", rnd)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheckBYOIPPrefix(t) },
```

Comment:
`testAccPreCheck` takes care of ensuring the credentials are in place
`testAccPreCheckAccount` ensures you have the account details setup

```suggestion
		PreCheck:  func() {
			testAccPreCheck(t)
			testAccPreCheckAccount(t)
			testAccPreCheckBYOIPPrefix(t) 
		},
```